### PR TITLE
Fix/template updates

### LINKED ISSUE #35: template fixes
- [x] drawer shows black on black text - define text color
- [x] tab titles not working properly on firefox? they disappear after clicking
- [x] analysis detail button doesn't work

### PR DESCRIPTION
**Pull request by acholyn**
Small template fixes, see issue for details, closes #35 
Improved conditional for video on list and detail views for film.
Removed "See in full" column header for film list - not in use; using "see more" at end of synopsis instead

### CODE REVIEW

File: mod_app/templates/analysis_list.html [hunk 1]
```diff
@@ -12,7 +12,9 @@ <h3>
     <div class="card-area">
         {% for analysis in analysis_list %}
             <div class="card card--analysis">
-                <h4 class="card__heading"><a href="{% url 'analysis_detail' pk=analysis.pk %}">{{analysis.title}}</a></h4>
+                <h4 class="card__heading"><a
+                    href="{% url 'analysis_detail' pk=analysis.pk %}"
+                >{{analysis.title}}</a></h4>
                 <div class="card__info">
                     <dl>
                         <dt>{{analysis.content|safe|truncatewords:50}} </dt>
```
Comment (tcouch) [inline mod_app/templates/analysis_list.html:20]: The width of dt elements appears to be constrained in normalise.css, which makes the text only take up about 2/3 of the width of the card. Is that what you want?

File: mod_app/templates/analysis_list.html [hunk 2]
```diff
@@ -29,7 +31,8 @@ <h4 class="card__heading"><a href="{% url 'analysis_detail' pk=analysis.pk %}">{
                             <dd>{{film.title}}</dd>
                         {% endfor %}
                     </dl>
-                    <button href="{% url 'analysis_detail' pk=analysis.pk %}"> view full details </button>
+                    <button>
+                        <a href="{% url 'analysis_detail' pk=analysis.pk %}"> view full details </a></button>
```
Comment (tcouch) [inline mod_app/templates/analysis_list.html:35]: It's possible to click the button, but not the <a> element. Could you just style the link as a button, or do something with the button onclick event?

File: mod_app/templates/tr_list.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+{% block content %}
+
+
+    <h3>
+        Teaching Resources available on this site
+    </h3>
+
+
+    <div class="card-area">
+        {% for tr in tr_list %}
```
Comment (tcouch) [inline mod_app/templates/tr_list.html:13]:
```suggestion
        {% for tr in teachingresources_list %}
```

**Review by tcouch — APPROVED** [review]
Just one minor fix

File: mod_app/templates/analysis_list.html (hunk 1 above)
Comment (acholyn) [reply mod_app/templates/analysis_list.html:20]: Potentially not but I'm also not sure they'll want to keep the cards; they may prefer a table for these as well. I'll leave it for now but make a note

**Pull request merged**